Example/ImportExample01.md

```diff
@@ -0,0 +1,91 @@
+## An example
+
+Imagine you have your own Frost-Server for sensor measurements. Unfortunately you have your old data stored in csv and excel files and now you want them online on your Frost-Server.
+Thats the perfect usecase for ChillImport. This software provides you an easy to use GUI to select and import your old sensor data.
+
+First download and install the program.
+
+The csv-file you want to import is: Examplefile.csv
+
+### Step 1: 

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-heading-punctuation] Don’t add a trailing `:` to headings](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)